holoviews/operation/datashader.py

```diff
@@ -1264,9 +1264,13 @@ def _process(self, element, key=None):
         kdims = element.kdims
 
         overrides = dict(self.p.items())
-        if 'normalization' in overrides and config.future_deprecations:
-            self.param.warning("Shading 'normalization' parameter deprecated, "
-                               "use 'cnorm' parameter instead'")
+        if 'normalization' in overrides:
+            if 'cnorm' in overrides:
+                self.param.warning("Cannot supply both 'normalization' and 'cnorm' for shading; "
+                                   "use 'cnorm' instead'")
```

Comment:
What do you mean 'cannot'...this is just a warning so something will happen and one will take precedence!

Reply:
In addition, it doesn't seem like an error if both specify the same value (e.g 'eq_hist`) even if it is redundant. A warning makes sense here but I don't think the wording is correct.

Reply:
Maybe "Both the 'cnorm' and 'normalization' keywords specified; 'cnorm' value taking precedence over deprecated 'normalization' option"